pygmt/src/plot.py

```diff
@@ -45,6 +45,7 @@
 )
 @kwargs_to_strings(R="sequence", c="sequence_comma", i="sequence_comma", p="sequence")
 @deprecate_parameter("sizes", "size", "v0.4.0", remove_version="v0.6.0")
+@deprecate_parameter("columns", "incols", "v0.4.0", remove_version="v0.6.0")
```

Comment:
The problem is that `test_plot_deprecate_columns_to_incols` uses a list instead of str for the `columns` parameter. The order should be:

1. Rename 'columns' to 'incols' via `@deprecate_parameter`
2. Convert long alias 'incols' to GMT short alias 'i' via `@use_alias`
3. Let the list type argument `[0, 1]` be converted to str type `0,1` via `@kwargs_to_strings`
4. Pass in `-i0,1` to `lib.call_module("plot", ...)`

Reply:
Thanks for your replies and the clarification @seisman and @weiji14.